config/deploy.yml

```
@@ -0,0 +1,39 @@
+service: campsite-sync-server
+
+image: julii776/campsite
+
+servers:
+  web:
+    - 35.154.218.222  
+
+env:
+  secret:
+    - API_BASE_URL
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)